helpers/lib/cdnify.js

```diff
@@ -76,8 +83,8 @@ module.exports = globals => {
             path = path.slice(1, path.length);
         }
 
-        if (path.match(/^assets\//)) {
-            path = path.substr(7, path.length);
+        if (path.startsWith('assets/')) {
+            path = path.slice(7, path.length);
```

Comment:
We should probably just omit this second parameter for clarity, and for any similar usages in this file.

Reply:
done.